Dockerfile

```
@@ -1,12 +1,27 @@
-FROM adoptopenjdk:11-jre-hotspot
-
-RUN apt-get update \
-    && apt-get install -y unzip \
-    && rm -rf /var/lib/apt/lists/*
+ARG VERSION=3.6
+ARG JAVA_VERSION=11-jre-hotspot
 
+FROM adoptopenjdk:${JAVA_VERSION}
 ARG VERSION
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

```suggestion
SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
```

Adds additional error handling we could have

---

Maybe it's safer to not add `-eu`, because those might be too strict.

---

I have always used `SHELL ["/bin/bash", "-o", "pipefail", "-c"]` - I think Hadolint prefers it as well.